package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@textlint-rule/textlint-rule-no-unmatched-pair",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

```suggestion
  "version": "2.0.2",
```
You do no need to chage version in PR.